bocas/artifacts/keras_history.py

```diff
@@ -1,16 +1,26 @@
-import pickle
-
 from bocas.artifacts.artifact import Artifact
+from bocas.artifacts.yaml_utils import parse_yaml_node
 
 
 class KerasHistory(Artifact):
     """KerasHistory is used to return the keras `model.fit()` history."""
 
+    yaml_tag = "!KerasHistory"
+
     def __init__(self, history, **kwargs):
         super().__init__(**kwargs)
-        self.history = history.history
+        self.history = history
 
     @property
     def metrics(self):
         """Returns a dictionary holding the metrics from `fit()`."""
         return self.history
+
```

Comment:
Metrics will now need to reference history one level deeper

Reply:
If the result is saved and then loaded, `return self.history` is ok since 
```python
config.update({"history": self.history.history})
```

But this is definitely an issue if you are trying to access the `KerasHistory` object prior to saving.  One way around this is to change the code from:
```python
history = model.fit(...)
bocas.artifacts.KerasHistory(history, name="history")
```
to 
```python
history = model.fit(...)
bocas.artifacts.KerasHistory(history.history, name="history")
```
but that might be confusing.

Reply:
Also could do a type check on the parameter

Reply:
yeah that looks nice, I like that

Reply:
Incorporated this in the newest version